src/utils/libfabric/libfabric_topology.cpp

```
@@ -195,6 +195,50 @@ nixlLibfabricTopology::isValidDevice(const std::string &efa_device) const {
     return std::find(all_devices.begin(), all_devices.end(), efa_device) != all_devices.end();
 }
 
+nixl_status_t
+nixlLibfabricTopology::hasPcieDevices(bool &device_found) const {
+    // Get EFA device info with PCIe addresses from libfabric
+    struct fi_info *hints, *info;
```

Cppcheck (2.20.0)

[style] The function 'getMetadata' is never used.

(unusedFunction)